SocialCareCaseViewerApi/V1/Boundary/Requests/CreatePersonalRelationshipRequest.cs

```diff
@@ -45,6 +48,8 @@ public CreatePersonalRelationshipRequestValidator()
                 .Matches("(?i:^Y|N)$").WithMessage("'isInformalCarer' must be 'Y' or 'N'.");
             RuleFor(pr => pr.Details)
                 .MaximumLength(1000).WithMessage("'details' must be less than or equal to 1,000 characters.");
+            RuleFor(pr => pr.CreatedBy)
+                .NotNull().WithMessage("'createdBy' must be provided.");
```

Comment:
Should we also be confirming that createdBy is an email

Reply:
It would probably also be a good idea in the usecase PersonalRelationshipsUseCase#ExecutePost to confirm the createdBy email is a worker in the system, otherwise throw `WorkerNotFoundException` or something you think is sensible

Reply:
Ahhh these are very good shouts, thanks!

Reply:
Updated so the use case checks and the API endpoint responds with 400.

Reply:
And updated so we check the `createdBy` in the request is an email address! (Totally didn't nearly forget about that! Cool we can do this so easily with FluentValidation. 😎)